src/main/java/com/github/fabienrenaud/jjb/model/Users.java

```diff
@@ -140,6 +148,182 @@ public int hashCode() {
         public String toString() {
             return "JsonDataObj{" + "_id=" + _id + ", index=" + index + ", guid=" + guid + ", isActive=" + isActive + ", balance=" + balance + ", picture=" + picture + ", age=" + age + ", eyeColor=" + eyeColor + ", name=" + name + ", gender=" + gender + ", company=" + company + ", email=" + email + ", phone=" + phone + ", address=" + address + ", about=" + about + ", registered=" + registered + ", latitude=" + latitude + ", longitude=" + longitude + ", tags=" + tags + ", friends=" + friends + ", greeting=" + greeting + ", favoriteFruit=" + favoriteFruit + '}';
         }
+
+        public String getId() {
+            return _id;
+        }
+
+        public void setId(String _id) {
+            this._id = _id;
+        }
+
+        public int getIndex() {
+            return index;
+        }
+
+        public void setIndex(int index) {
+            this.index = index;
+        }
+
+        public String getGuid() {
+            return guid;
+        }
+
+        public void setGuid(String guid) {
+            this.guid = guid;
+        }
+
+        public boolean isActive() {
```

Comment:
@rbygrave I'd like to ask you for some help as you added the avaje-jsonb library.

I try to add qson in this PR which requires getters and setters. But when these are present, the tests for avaje-jsonb are failing.

**com.github.fabienrenaud.jjb.databind.UsersDeserializationTest::avajeJsonb_diesel**
```
Unexpected end of JSON input
io.avaje.jsonb.JsonDataException: Unexpected end of JSON input
	at app//io.avaje.jsonb.stream.JParser.read(JParser.java:230)
```

**com.github.fabienrenaud.jjb.stream.UsersDeserializationTest::avajeJsonb_diesel**
```
Expecting '}' as object end
io.avaje.jsonb.JsonDataException: Expecting '}' as object end
	at app//io.avaje.jsonb.stream.JParser.newParseError(JParser.java:971)
```

The culprit seems to be the getter/setter for `isActive`.
Do you have an idea what could be the issue and how to fix it?
Or is there a way to enforce using field access even though getters/setters are available?

Reply:
Yes, I'll have a look later this weekend. My suspicion is that it isn't expecting and handling both field and method access ... so I should be able to reproduce and fix.

Reply:
Ok I'm pretty sure I have a handle on the problem. We have a field named `isActive` (so the json payload coming in also has that as the property name).  In adding the getter/setter of `isActive()` & `setActive()` this implies that the "property" is now called `active` rather than `isActive` - the json payload coming in has `"active"` as the key. To jsonb this is an unnamed property as it is still expecting `isActive` to be used, avajeJsonb_diesel has a bug here in that is not skipping that value properly.

So I can look to fix that bug in avajeJsonb_diesel. 

The other factor I'm pondering is that to me it looks like some json parsers are going to interpret the property name as `isActive` based on the public field and some might now interpret the property name to instead be `active` based on the getters and setters (but with lenient parsing unknown properties are being skipped and not failing the test per say).

Reply:
I've submitted PR - https://github.com/fabienrenaud/java-json-benchmark/pull/54 which bumps avaje-jsonb to 1.0-RC4.  Note that this bug was fixed in version 0.12 but 1.0-RC4 is the latest.

You can similarly bump avaje-jsonb to 1.0-RC4 in this PR or if that PR gets merged then rebase with that change.

Also note that with both public fields _AND_ getters/setters there is some variance with how that is treated by the various libraries (for example DSL will generate both `isActive` and `active`). For avaje-jsonb I've gone with getters/setters have precedence over public fields.

Reply:
The quick and detailed response is well appreciated.